src/rpm/package.rs

```diff
@@ -178,40 +178,40 @@ impl Package {
         Ok(())
     }
 
-    /// Return the key id (issuer) of the signature as a hexadecimal string
+    /// Return the key ids (issuers) of the signature as a hexadecimal string
     #[cfg(feature = "signature-pgp")]
-    pub fn signature_key_id(&self) -> Result<Option<String>, Error> {
+    pub fn signature_key_ids(&self) -> Result<Vec<String>, Error> {
```

Comment:
`rpgp` changed the underlying API here to reflect that signatures can apparently have more than one issuer / key ID.